src/test/java/io/stargate/sgv2/jsonapi/testresource/StargateTestResource.java

```diff
@@ -0,0 +1,264 @@
+//
```

Comment:
This is temporary right? Are we planning to remove this class copied from main stargate repo prior to merging? (Presumably because Stargate version is updated with what we need)

Reply:
Yes., Will remove this once the stargate api release is done.